skycultures/tibetan/description.en.utf8

```diff
@@ -0,0 +1,129 @@
+The following table is from Cornu, with photographs showing the figures illustrated in the circular platform. Note the unexpected appearance of Cancer. Is it a frog?
+</p>
+
+<table>
+<th><td>  </td><td>IAU</td><td>Tibetan</td><td>Indian </td><td>                                                    </td></th>
+<tr><td> 1</td><td>Ari</td><td>Luk    </td><td>Meṣa   </td><td><img src="StagPhu_01_Ari.jpg" height=150 alt="Ari"/></td></tr>
+<tr><td> 2</td><td>Tau</td><td>Lang   </td><td>Vṛṣa   </td><td><img src="StagPhu_02_Tau.jpg" height=150 alt="Tau"/></td></tr>
+<tr><td> 3</td><td>Gem</td><td>Trik   </td><td>Mithuna</td><td><img src="StagPhu_03_Gem.jpg" height=150 alt="Gem"/></td></tr>
+<tr><td> 4</td><td>Cnc</td><td>Karkata</td><td>Karka  </td><td><img src="StagPhu_04_Cnc_Frog.jpg" height=150 alt="Cnc"/></td></tr>
+<tr><td> 5</td><td>Leo</td><td>Senge  </td><td>Simha  </td><td><img src="StagPhu_05_Leo.jpg" height=150 alt="Leo"/></td></tr>
+<tr><td> 6</td><td>Vir</td><td>Pumo   </td><td>Kanyā  </td><td><img src="StagPhu_06_Vir.jpg" height=150 alt="Vir"/></td></tr>
+<tr><td> 7</td><td>Lib</td><td>Sangwa </td><td>Tulā   </td><td><img src="StagPhu_07_Lib.jpg" height=150 alt="Lib"/></td></tr>
+<tr><td> 8</td><td>Sco</td><td>Dikpa  </td><td>Vṛscika</td><td><img src="StagPhu_08_Sco.jpg" height=150 alt="Sco"/></td></tr>
+<tr><td> 9</td><td>Sgr</td><td>Zhu    </td><td>Dhanus </td><td><img src="StagPhu_09_Sgr.jpg" height=150 alt="Sgr"/></td></tr>
+<tr><td>10</td><td>Cap</td><td>Chusin </td><td>Makara </td><td><img src="StagPhu_10_Cap.jpg" height=150 alt="Cap"/></td></tr>
+<tr><td>11</td><td>Aqr</td><td>Bumpa  </td><td>Kumbha </td><td><img src="StagPhu_11_Aqr.jpg" height=150 alt="Aqr"/></td></tr>
+<tr><td>12</td><td>Psc</td><td>Nya    </td><td>Mīna   </td><td><img src="StagPhu_12_Psc.jpg" height=150 alt="Psc"/></td></tr>
+</table>
+
+
+
+<h3>The Lunar Mansions</h3>
+
+<p>The Men Tsee Khang emblem shows 27 stick figures for the Lunar mansions arranged in a circle around the central figure. 
+The same figures are also laid out in cement with bright stones connected by black lines in the circular platform which has been erected when the observatory was rebuilt. 
+These 27 <em>gyukar</em> figures are described as being equal to the Indian <em>Nakṣatras</em> and represent sections of 13°20' longitude along the ecliptic. 
+Actually, one of the <em>gyukars</em> consists of 2 constellations, therefore 28 <em>lunar constellations</em> are named.</p>
+
+
+<p>However, not much is known of the exact identification of the Tibetan Lunar Stations outside of Tibet. 
+Most literature is Tibetan or Chinese only, and there were no Tibetan star maps available. Also our guides could not explain any of the following to us.</p>
+
+
+<p>The following table was taken from Cornu. Note that Tibetans start counting at zero, while the Indians count from 1. </p>
+
+<table>
```

Comment:
Please remove extra spaces in the table

Reply:
Or mark the text as untranslatable if it is necessary

Reply:
It will be nightmare for translators after extract the text because every space will be added as translatable part of string

Reply:
Is it possible to apply "trim leading and trailing whitespace" after extraction? Makes life easier for authors and translators.

Reply:
For gettext/xgettext the lines with and without leading/trailing whitespaces are different lines. It’s limitation by design

Reply:
I have updated the first table to show what I meant with above question: Is it acceptable to at least align the opening td, and leave space between closing td and the next opening td? Or is the empty space between td elements a problem?

Reply:
Adding whitespaces between </td><td> are acceptable - this part was ignored by extraction tool